docs/source/model_design/general_research.rst

```diff
@@ -831,11 +831,93 @@ Ecologic Studies
 Meta-Analyses and Meta-Regressions
 ++++++++++++++++++++++++++++++++++
 
+Meta-Analyses and Meta-Regressions are two similar study designs that allow researchers to combine and summarize findings from several individual studies that investigated the same association between an exposure and outcome. Studies included in meta-analyses and meta-regressions are typically identified using systematic literature searchers (see `Conducting Systematic Literature Searches/Reviews`_) and chosen for inclusion based on a set of inclusion and exclusion criteria for that particular meta-analysis or -regression. While each study in a meta-analysis or meta-regression should have the same exposure and outcome of interest, they may differ with respect to study design, study populations, and confounding bias. Briefly, meta-analysis summarizes study findings without formal control for differences between individual studies while meta-regression allows for flexible formal control for these differences. Details are discussed below.
```

Comment:
I am still not entirely clear on how to do a meta analyses without meta regression (would you take a weighted average? is there a technique for doing this with random effects without meta regression?) but as far as I understand it, meta-analyses are a type of study, and meta regressions are a tool used in meta analyses, (as opposed to them being similar/distinct study designs).